bokehjs/src/coffee/models/renderers/graph_renderer.coffee

```diff
@@ -0,0 +1,68 @@
+import {Circle} from "../glyphs/circle"
+import {MultiLine} from "../glyphs/multi_line"
+import {GraphDataSource} from "../graphs/graph_data_source"
+import {Renderer, RendererView} from "../renderers/renderer"
+
+import * as p from "core/properties"
+import {range} from "core/util/array"
+
+export class GraphRendererView extends RendererView
+
+  initialize: (options) ->
+    super(options)
+    @edges = @build_glyph_view(@model.edges)
+    @nodes = @build_glyph_view(@model.nodes)
+
+    # hackery
+    @edges._index_data = () -> null
```

Comment:
Right, using the glyphs as implementation details is a perfectly reasonable thing to want to do, but is also a thing that has not really *been done* before. So I would expect some changes might be necessary to accommodate this case, likely loosening some assumptions in various places.